src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -994,19 +996,26 @@ public String findMailAddressFor(User u) {
         // Make these available (private in parent class and no get methods!)
         String rolePrefix = "ROLE_";
         boolean convertToUpperCase = true;
+        LDAPGroupMembershipStrategy groupMembershipStrategy;
+        LdapUserSearch ldapSearch;
 
-        public AuthoritiesPopulatorImpl(InitialDirContextFactory initialDirContextFactory, String groupSearchBase) {
+        public AuthoritiesPopulatorImpl(InitialDirContextFactory initialDirContextFactory, String groupSearchBase, LDAPGroupMembershipStrategy groupMembershipStrategy) {
```

Comment:
ahhh but what about an installation that has a custom `LDAPBindSecurityRealm.groovy` that they haven't touched in years... with this change you will break their instance and they cannot login to figure out why